src/System.Net.Security/src/System/Net/FixedSizeReader.cs

```diff
@@ -46,14 +46,14 @@ public static int ReadPacket(Stream transport, byte[] buffer, int offset, int co
         /// Completes "request" with 0 if 0 bytes was requested or legitimate EOF received.
         /// Otherwise, reads as directed or completes "request" with an Exception.
         /// </summary>
-        public static async void ReadPacketAsync(Stream transport, AsyncProtocolRequest request) // "async Task" might result in additional, unnecessary allocation
+        public static async void ReadPacketAsync(Stream transport, AsyncProtocolRequest request, CancellationToken cancellationToken) // "async Task" might result in additional, unnecessary allocation
```

Comment:
As long as you're editing this line, you can remove the comment at the end; that's no longer the case.